Gopkg.toml

```diff
@@ -5,11 +5,11 @@
 
 [[constraint]]
   name = "github.com/ethereum/go-ethereum"
-  version = "v1.8.12"
+  version = "1.8.12"
 
 [[constraint]]
   name = "github.com/republicprotocol/republic-go"
-  branch = "nightly"
```

Comment:
Make sure you change this back to nightly when null-pointer-protection been merged into nightly